client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -169,6 +169,25 @@ function CheckoutSummaryPlanFeatures() {
 
 function getPlanFeatures( plan, translate, hasDomainsInCart ) {
 	if (
+		'personal-bundle' === plan.wpcom_meta?.product_slug ||
+		'personal-bundle-2y' === plan.wpcom_meta?.product_slug
+	) {
+		return [
+			! hasDomainsInCart && translate( 'Free domain for one year' ),
+			translate( 'Remove WordPress.com ads' ),
+			translate( 'Limit your content to paying subscribers.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Limit content to paying subscribers.' )` *ES Score: 9*